src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/zipfilestatus/ZipFileEnvelopeMapperTest.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.model.mapper.zipfilestatus;
+
+import org.assertj.core.api.SoftAssertions;
+import org.junit.Test;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.Envelope;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.Status;
+import uk.gov.hmcts.reform.bulkscanprocessor.model.out.zipfilestatus.ZipFileEnvelope;
+
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+
+public class ZipFileEnvelopeMapperTest {
+
+    @Test
+    public void should_map_envelope_properly() {
+        // given
+        Envelope input = mock(Envelope.class);
+        given(input.getId()).willReturn(UUID.randomUUID());
+        given(input.getZipFileName()).willReturn("x.zip");
+        given(input.getStatus()).willReturn(Status.PROCESSED);
+
+        // when
+        ZipFileEnvelope output = ZipFileEnvelopeMapper.fromEnvelope(input);
+
+        // then
+        SoftAssertions softly = new SoftAssertions();
```

Comment:
Is there any reason for using `SoftAssertions` rather than usual `Assert` statements?

Reply:
perhaps a practice rather than adaptation. unit tests is kind of "constant run" from developer. should fail fast until you solve the problem.

nevertheless - summary is good 👍